jhucourse/woj/backend/game.py

```diff
@@ -0,0 +1,134 @@
+'import json'
+import numpy
+import player
+'import QuestionBank'
+
+class game:
+    def __init__(self):
+        self.player_list = []
+        self.current_player = 0
+        self.spin_counter = 50
+        self.current_round = 1
+        self.turnPoints = 0
+        self.question_bank
+
+    'Add a player to the list'
+    def addPlayer(self, name):
+        Player = player(name)
+        self.player_list.append(Player)
+        return
+
+    'Remove a player from the list'
+    def deletePlayer(self, player):
+        self.player_list.remove(player)
+        return
+
+    'Get the next player in the list'
+    def nextPlayer(self):
+        len = self.player_list.count
```

Comment:
This isn't really necessary. We can replace everywhere we see len with self.player_list.count